code/dynamic_programming/src/fibonacci/fibonacci.cpp

```diff
@@ -1,28 +1,42 @@
-from sys import argv
+/* Part of Cosmos by OpenGenus Foundation */
 
+#include <string>
+#include <sstream>
+#include <iostream>
 
-def fibonacci(n):
-    """
-    N'th fibonacci number using Dynamic Programming
-    """
-    arr = [0, 1]  # 1st two numbers as 0 and 1
+using namespace std;
 
-    while len(arr) < n + 1:
-        arr.append(0)
+int dynamic_programming(int num);
```

Comment:
Can you aptly rename the function? Cause the function is used to find the nth fibonacci number, so it should have name which corresponds to its functionality rather than simply dynamic_programming in general.